### PR TITLE
[merged] fix bugs and migrate to .redhat-ci.yml

### PR DESCRIPTION
**Pull request by jlebon**
This patch add a new `.redhat-ci.yml` file. This file is read by the
internal Red Hat CI service to determine testing procedures. For more
information, see:

https://github.com/jlebon/redhat-ci

---

:warning: Do not merge yet. The backend is not yet completely hooked up. Just opening the PR ahead of time to get folks familiar with it. Once this is merged, I will be retiring the current PR tester. :warning:

### CODE REVIEW

**Comment by jlebon** [comment]
@aweiteka @runcom This could also provide a way to support more recent `skopeo` builds for testing. The YAML file supports an `extra-repos` [key](https://github.com/jlebon/redhat-ci/blob/master/sample.redhat-ci.yml#L30), so e.g. if someone sets up a COPR, it would be trivial to hook up here.

There's also the CAHC skopeo build, but there's the epoch mismatch issue, and I'm not sure if all the test requirements are available on CentOS, though you're free to explore that area as well.

File: .redhat-ci.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+host:
+  distro: fedora/24/cloud
```
Comment (cgwalters) [inline .redhat-ci.yml]: Same complaint as https://github.com/projectatomic/commissaire/pull/16#pullrequestreview-3143852 :smile:

**Comment by jlebon** [comment]
Bug fixes and full migration to Atomic Host tests. ^

**Comment by rhatdan** [comment]
LGTM

**Comment by jlebon** [comment]
Could we review & merge this soon? I'd like to get it in before things get out of sync and start gating PRs on the AH tests.

**Comment by cgwalters** [comment]
Long term, I think we should split the test suite into both traditional "make check" (uid != 0, uninstalled), and "destructive system tests".  

The groff etc would run under `make check` just fine in a container.  The destructive system tests would run docker.

**Comment by cgwalters** [comment]
@rh-atomic-bot r+ d7024bb

**Comment by jlebon** [comment]
Right, in fact as it stands right now, the whole testsuite could probably be run from a docker container with the socket mounted, except for the migrate and storage tests. Shouldn't be too hard to modify.

I think I'll leave the previous PR tester since it runs on Python 2. At least until we make redhat-ci support multiple testsuites. In the meantime, can we make homu gate on the new context too?

**Pull request closed without merging**